source/slang/slang-emit-spirv.cpp

```diff
@@ -6453,35 +6457,21 @@ struct SPIRVEmitContext
         else if (auto ptrType = as<IRPtrTypeBase>(type))
         {
             IRType* baseType = ptrType->getValueType();
-            if (as<IRBasicType>(baseType))
-            {
-                // If the base type of the pointer is basic types,
-                // emit the basic types and emit DebugTypePointer with it.
-                SpvInst* debugBaseType = emitDebugType(baseType);
-                SpvStorageClass storageClass = SpvStorageClassFunction;
-                if (ptrType->hasAddressSpace())
-                    storageClass = addressSpaceToStorageClass(ptrType->getAddressSpace());
 
-                return emitOpDebugTypePointer(
-                    getSection(SpvLogicalSectionID::ConstantsAndTypes),
-                    nullptr,
-                    m_voidType,
-                    getNonSemanticDebugInfoExtInst(),
-                    debugBaseType,
-                    builder.getIntValue(builder.getUIntType(), storageClass),
-                    builder.getIntValue(builder.getUIntType(), kUnknownPhysicalLayout));
-            }
+            // If the base type of the pointer is basic types,
```

Comment:
I think this comment should be removed, because the "if" condition it was referring to is removed.